.semaphore/semaphore.yml

```
@@ -0,0 +1,25 @@
+version: v1.0
+name: AppSignal mio_worker
+agent:
+  machine:
+    type: e1-standard-2
+  containers:
+    - name: main
+      image: registry.semaphoreci.com/rust:1.51
+blocks:
+  - name: 'Default'
+    task:
+      prologue:
+        commands:
+          - checkout
+      jobs:
+        - name: Formatter and linter
+          commands:
+            - rustup component add rustfmt clippy
```

```suggestion
            - rustup component add clippy
```
I think rustfmt is installed by default already.